examples/vite/src/eyepop.min.js

```diff
@@ -0,0 +1 @@
+/Users/edmunddao/_SPACE/EyePop.ai/Untitled/eyepop-sdk-node/src/eyepop/dist/eyepop.min.js
```

Comment:
what are those files?

Reply:
They were symlinks used for testing - I removed them and added them directly to the server

`    resolve: {
        alias: {
            'eyepop': path.resolve(__dirname, '../../src/eyepop/dist/eyepop.min.js'),
            'eyepop-render-2d': path.resolve(__dirname, '../../src/eyepop-render-2d/dist/eyepop.render2d.min.js'),
            'example.jpg': path.resolve(__dirname, '../../examples/example.jpg'),
            'large_example.jpg': path.resolve(__dirname, '../../examples/large_example.jpg'),
        }
    },`

https://github.com/eyepop-ai/eyepop-sdk-node/pull/31/commits/1b14f6251cf82dc803abbcdb14a2710bb675b917